src/pages/AnimeKO/main.ts

```diff
@@ -0,0 +1,82 @@
+import { pageInterface } from '../pageInterface';
+
+let jsonData;
+
+export const AnimeKO: pageInterface = {
+  name: 'AnimeKO',
+  domain: 'https://animeko.co',
+  languages: ['French'],
+  type: 'anime',
+  isOverviewPage() {
+    return !!j.$('.showcase-shadow').length;
+  },
+  isSyncPage() {
+    return !!j.$('.player-wrap').length;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('.small-card h2').text().trim();
+    },
+    getIdentifier(url) {
+      return utils
```

Comment:
Here it is probably better to use `return AnimeKO.sync.getIdentifier(AnimeKO.overview!.getOverviewUrl(url))` to follow DRY.

Reply:
this throws an error of "Property 'getOverviewUrl' does not exist"

Reply:
Was the other way around
`return AnimeKO.overview!.getIdentifier(AnimeKO.sync.getOverviewUrl(url))`